department-of-veterans-affairs/caseflow
# Bsampson/update seed data

### Description
Added more tasks to substitutions seed data. Also changed the name and ssn for appeals from sanitized json to be more obvious that this is not prod data.

### Acceptance Criteria
- [x] Active and completed tasks have been added to substitution seed data.

### Testing Plan
1. Login in as cob admin or user.
2. search for appeal '54545454'
3. open a dispatched case and check that there are open and completed tasks.
4. open a pending case and check that there are open and completed tasks.
